lagunita/lms/templates/static_templates/copyright.html

```diff
@@ -9,16 +9,18 @@
 <%namespace name='static' file='/static_content.html'/>
 <%namespace name='theme_settings' file='/theme-settings.html'/>
 
-<%block name="pagetitle">${_('Copyright Complaints')}</%block>
-
 <%
   theme_vars = theme_settings.get_theme_vars()
   effective_date = theme_vars['effective_date']
   contact_email = '<a href="mailto:{email}">{email}</a>'.format(email=settings.CONTACT_EMAIL)
 %>
 
+<%block name="pagetitle">${_('Copyright Complaints')}</%block>
```

Comment:
what I wish this could be would be something like:

```mako
<%
   tos_links = theme_settings.get_tos_links()
%>

<%block name="pagetitle">${tos_links['copyright']}</%block>
```